web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -72,68 +82,98 @@ function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
     setOrder(order === 'asc' ? 'desc' : 'asc')
   }
 
+  interface TableHeaderProps {
+    title: string
+    testId?: string
+  }
+
+  const TableHeader = (tableHeaderProps: TableHeaderProps) => {
+    const tableHeaderClasses = useStyles()
+    return (
+      <Toolbar data-testid={`${tableHeaderProps.testId}-header`} disableGutters={true}>
+        <Typography className={tableHeaderClasses.title}>
+          {tableHeaderProps.title}
+        </Typography>
+        <Tooltip title={open ? 'Collapse table' : 'Expand table'}>
+          <IconButton
+            aria-label={open ? 'collapse table' : 'expand table'}
+            size="small"
+            onClick={() => setOpen(!open)}
+            data-testid={`${tableHeaderProps.testId}-collapse`}
+          >
+            <KeyboardArrowUpIcon
+              className={clsx(
+                !open && tableHeaderClasses.pointDown,
+                open && tableHeaderClasses.pointUp
+              )}
+            />
+          </IconButton>
+        </Tooltip>
+      </Toolbar>
+    )
+  }
+
   return (
     <div className={classes.display} data-testid={props.testId}>
-      <Typography className={classes.title} component="div" variant="subtitle2">
-        {props.title}
-      </Typography>
-
+      <TableHeader title={props.title} testId={props.testId}></TableHeader>
       <Paper className={classes.paper} elevation={1}>
         <TableContainer className={classes.tableContainer}>
-          <Table stickyHeader size="small" aria-label="sortable wx table">
-            <TableHead>
-              <TableRow>
-                {props.columns.map(column => {
-                  const canSort = column.id === 'datetime'
-
-                  return (
-                    <TableCell
-                      key={column.id}
-                      align={column.align}
-                      style={{ minWidth: column.minWidth, maxWidth: column.maxWidth }}
-                      sortDirection={canSort ? order : false}
-                    >
-                      {canSort ? (
-                        <TableSortLabel
-                          active={canSort}
-                          direction={order}
-                          onClick={toggleDatetimeOrder}
-                        >
-                          {column.label}
-                        </TableSortLabel>
-                      ) : (
-                        column.label
-                      )}
-                    </TableCell>
-                  )
-                })}
-              </TableRow>
-            </TableHead>
-
-            <TableBody>
-              {rowsSortedByDatetime.map((row, idx) => (
-                <TableRow key={idx} hover tabIndex={-1}>
+          <Collapse in={open} timeout="auto" unmountOnExit>
```

Comment:
The "auto" timeout looks a bit weird to me (seems too slow) - have you tried with explicit low values?

Reply:
Sped it up to 250 ms.